controllers/clustergroupupgrade_controller.go

```diff
@@ -439,19 +583,19 @@ func (r *ClusterGroupUpgradeReconciler) ensureBatchPlacementRules(ctx context.Co
 		}
 	}
 
-	return placementRules, nil
+	return &placementRule, nil
 }
 
-func (r *ClusterGroupUpgradeReconciler) newBatchPlacementRule(ctx context.Context, clusterGroupUpgrade *ranv1alpha1.ClusterGroupUpgrade, batch []string, batchIndex int) (*unstructured.Unstructured, error) {
+func (r *ClusterGroupUpgradeReconciler) newBatchPlacementRule(ctx context.Context, clusterGroupUpgrade *ranv1alpha1.ClusterGroupUpgrade, managedPolicy *unstructured.Unstructured) (*unstructured.Unstructured, error) {
 	u := &unstructured.Unstructured{}
 	u.Object = map[string]interface{}{
 		"metadata": map[string]interface{}{
-			"name":      clusterGroupUpgrade.Name + "-batch" + strconv.Itoa(batchIndex),
+			"name":      clusterGroupUpgrade.Name + "-" + managedPolicy.GetName(),
```

Comment:
Same name calculation here and below. Maybe create the name in a var and use it both places?

Reply:
Yes, I also thought about that. Will do.

Reply:
Done